ui.go

```diff
@@ -383,39 +383,26 @@ func (ui *UI) renew() {
 }
 
 func (ui *UI) loadFile(nav *Nav) {
-	dir := nav.currDir()
-
-	if len(dir.fi) == 0 {
+	curr, err := nav.currFile()
+	if err != nil {
 		return
 	}
 
-	curr := nav.currFile()
-
 	ui.message = fmt.Sprintf("%v %v %v", curr.Mode(), humanize(curr.Size()), curr.ModTime().Format(time.ANSIC))
```

Comment:
I don't mind this very much. Until me or someone else is bothered enough to implement an option for this (e.g. `followlinks`) we can leave it as it is I guess.